### DISCUSSION TITLE
There is no DeDRM_plugin.zip

### DISCUSSION THREAD

**Discussion by CandyBars65** [noDRM/DeDRM_tools#550] [General]
Seriously, DeDRM_plugin.zip doesn't exist in the archive, but all the readme's and faq's reference it when adding the plugin to calibre.  Extremely frustrating!

**Comment by ElleKayEm** [comment]
There is.  Go here: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9

Download the first file under Assets.  Unzip it.  Make sure your browser isn't auto unzipping everything.  You will have 5 files: the 2 plugin zip files and 3 readme text files.

**Reply by CandyBars65** [reply]
Thank you!  This site is madness to navigate, I never even saw that!

**Reply by usernamemiv** [reply]
thanks i had the same problem :D

**Comment by sometimescasey** [comment]
Posting in case other folks have the same confusion as me, because it's not obvious (even if you have some idea of what a `__init__.py` is):

Download and unzip this (the **release**, from https://github.com/noDRM/DeDRM_tools/releases):

<img width="1268" alt="image" src="https://github.com/user-attachments/assets/a6588ec3-b193-4467-b6cc-d911f6c6ab5e" />

Do not download and unzip the repo itself, or the source code from the releases page. (Confusingly the latter will download with almost the exact same filename as the release):

<img width="459" alt="image" src="https://github.com/user-attachments/assets/53c006c4-f5ba-4c63-9b5c-485bce3a9e2f" />

<img width="459" alt="image" src="https://github.com/user-attachments/assets/c8c6f29c-2bff-4280-8e09-4b58c7622dec" />

If you get the `It does not contain a top-level __init__.py file` error you have either 1) downloaded the release but forgot to unzip it and select the .zip inside; or 2) downloaded the repo:

<img width="495" alt="image" src="https://github.com/user-attachments/assets/93dd3ef2-db6b-4573-941f-5d1059e02a48" />

**Reply by Maxine025** [reply]
Thank you so much sometimescasey!!! including the pics made my day :)
